playwright-helpers/.eslintrc.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
why do you have playwright relateds files in this PR ?

Reply:
Because playwright is in the generator and the setup needs to be updated